src/modules/context/context-api-pixie.js

```diff
@@ -19,20 +17,19 @@ import {
   listRecoveryQuestionChoices,
   loginRecovery2
 } from '../login/recovery2.js'
-import { makeCoreRoot, startCoreRoot } from '../root.js'
+import type { RootProps } from '../rootPixie.js'
 import { awaitPluginsLoaded } from '../selectors.js'
 
-export function makeContext (opts: AbcContextOptions) {
-  const coreRoot = makeCoreRoot(opts)
-  startCoreRoot(coreRoot)
-  const { redux } = coreRoot
-
-  const appId =
-    opts.appId != null
-      ? opts.appId
-      : typeof opts.accountType === 'string'
-        ? opts.accountType.replace(/^account.repo:/, '')
-        : ''
+export const contextApiPixie = (input: PixieInput<RootProps>) => (
+  props: RootProps
+) => {
+  input.onOutput(makeContext(input))
+  return stopUpdates
+}
+
+export function makeContext (propsWrapper: PixieInput<RootProps>) {
```

Comment:
Why is this exported? Who imports this makeContext? From this file, it looks like makeContext changed from being a function that accepts an AbcContextOptions to one that takes PixieInput<RootProps>. But no other code changed to complement this switch.